Update set-up-windows-containers-with-powershell.md

stackoverjoe opened the pull request:
## Summary

The example Dockerfile did not work. Updated to get tag ltsc2019 which will be installed with powershell 7, so "RUN powershell.." switched to "RUN pwsh..".

stackoverjoe committed:
Update set-up-windows-containers-with-powershell.md

The example Dockerfile did not work. Updated to get LTSC which will be installed with powershell 7, so "RUN powershell.." switched to "RUN pwsh..".

sughosneo commented:
Hi @stackoverjoe, thank you for submitting this PR. 

If you use `FROM mcr.microsoft.com/windows/servercore:ltsc2019` then by default it uses PowerShell version 5. 

![image](https://user-images.githubusercontent.com/13281246/96564667-17baed00-12e1-11eb-80a7-b6b271ca83d1.png)

And that's the reason while running the step `RUN pwsh -Command Add-WindowsFeature Web-Server` fails with following error. 

![image](https://user-images.githubusercontent.com/13281246/96564631-0a9dfe00-12e1-11eb-9338-f1c3d2a23e6c.png)

So it's better to keep the `RUN powershell ` step as is.

```Dockerfile
FROM mcr.microsoft.com/windows/servercore:ltsc2019
LABEL Description="IIS" Vendor="Microsoft" Version="10"
RUN powershell -Command Add-WindowsFeature Web-Server
CMD [ "ping", "localhost", "-t" ]
```

And that can be run successfully.

![image](https://user-images.githubusercontent.com/13281246/96565304-d4ad4980-12e1-11eb-9e36-ec9d11f12d66.png)

Reference 

https://devblogs.microsoft.com/dotnet/we-made-windows-server-core-container-images-40-smaller/

stackoverjoe commented:
Hi @sughosneo thank you for the reply.

I was mistaken by thinking that mcr.microsoft.com/powershell installed the same image on windows! Thank you for the correction!

stackoverjoe committed:
Update set-up-windows-containers-with-powershell.md

Return pwsh command to powershell as mcr.microsoft.com/windows/servercore:ltsc2019 installs powershell 5 by default, not 7.

sughosneo commented:
> Hi @sughosneo thank you for the reply.
> 
> I was mistaken by thinking that mcr.microsoft.com/powershell installed the same image on windows! Thank you for the correction!

Looks good ! Thank you
